cunumeric/module.py

```diff
@@ -6227,3 +6227,719 @@ def bincount(
             )
             out._thunk.bincount(x._thunk, weights=weights._thunk)
     return out
+
+
+# account for 0-based indexing
+# there's no negative numbers
+# arithmetic at this level,
+# (pos, k) are always positive!
+#
+def floor_i(k):
+    j = k - 1 if k > 0 else 0
+    return int(j)
+
+
+# Generic rule: if `q` input value falls onto a node, then return that node
+
+# Discontinuous methods:
+#
+# 'inverted_cdf'
+# q = quantile input \in [0, 1]
+# n = sizeof(array)
+#
+# 'inverted_cdf'
+# q = quantile input \in [0, 1]
+# n = sizeof(array)
+#
+def inverted_cdf(q, n):
+    pos = q * n
+    k = floor(pos)
```

Comment:
Let's use `math.floor` (which only works with scalars) in these quantile methods. We've imported `floor` from `._ufunc.floating`, which again goes through the whole Legion machinery, and that's not necessary for scalar values.

Reply:
Done.